readme.md

```diff
@@ -2,109 +2,106 @@
-* **Write your code DRY** and **build your APIs RESTful**.
-* **Be consistent with your code style.** You're working in teams, but you're only making one app per team. Make sure it looks like a unified effort.
-* **Commit early, commit often.** Don’t be afraid to break something because you can always go back in time to a previous version.
-* **Keep user stories small and well-defined**, and remember – user stories focus on what a user needs, not what development tasks need accomplishing.
-* **Write code another developer wouldn't have to ask you about**. Do your naming conventions make sense? Would another developer be able to look at your app and understand what everything is?
-* **Make it all well-formatted.** Are you indenting, consistently? Can we find the start and end of every div, curly brace, etc?
-* **Comment your code.** Will someone understand what is going on in each block or function? Even if it's obvious, explaining the what & why means someone else can pick it up and get it.
-* **Write pseudocode before you write actual code.** Thinking through the logic of something helps.
+---
 
+### Evaluation and Submission
 
----
+Each member of the group must submit a pull request to the `pbj-project3` repository by *12:30 pm, Friday, August 28th**.
 
-### Potential Project Ideas
+[See the requirements for the contents of the submission here.](evaluation.md)
 
-For this project, we want you to work with your team to build a creative product that you actually think someone will want to use. We won't have time to do tons of customer research, but take some time to brainstorm. If you're struggling for ideas, the ones below could help get you started.
+Feedback will be returned to you by **9:00 am, Monday, August 31st**.
 
-##### Bucketli.st
-Besides finishing WDI, you surely have one or two things you'd love to do with your life. Let's get 'em on paper! You could integrate with a third-party location-based API to allow users to search for a location or venue to add to their bucket list items.
+Note that instructors will *not* be grading on a numbered scale this time. Projects will simply be either "meets expectations" or "does not meet expectations".
+
+---
 
-##### Survey App
-Imagine sending out a survey to everyone in the class – what should we eat for lunch today? Or 1-5, how well did you understand what we just learned? It would be even more awesome if it were realtime, so you could see answers pouring in as they're submitted.
+### Attendance
 
-##### Hello, Comments
-Imagine the benefits of having an API where you could embed comments into any website you want. They could even update in realtime if you wanted, so that you'd never have to refresh the page. CMS providers across the world could quit writing code from scratch and just embed your widget instead.
+Attendance will be taken at 9:00am on Tuesday, and at 12:00pm on Wednesday. Groups are otherwise welcome to work outside of GA at their collective discretion.
```

Comment:
i would actually vote for an instructor scrum with each group on wednesday. But I also like student discretion here.